packages/xo-server/config.toml

```diff
@@ -29,6 +29,9 @@ guessVhdSizeOnImport = false
 # be turned for investigation by the administrator.
 verboseApiLogsOnErrors = false
 
+# timeout before reporting servers issues
```

Comment:
Be more precise in your comments:

`# if no events could be fetched during this delay, the server will be marked as disconnected`